package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cozy-proxy",
-  "version": "2.1.3",
+  "version": "2.1.4",
```

Comment:
Why are you upgrading the application version ? Usually we only do this before publishing the application.